### PR TITLE
Properly persist clearing the Red Hat disabled rule alert

### PR DESCRIPTION
**Pull request by mhuth**
Actually works this time.   Whodathunk 'false' === true !?!  Programming101 fail for Mark :facepalm: 

But anyway, should I use localStorage instead of sessionStorage though?  Then the alert will never be back :terminator:

### CODE REVIEW

**Review by LI0131 — COMMENTED** [review]
Looking good @mhuth 😁 You might consider using `JSON.parse` to deserialize the value from the sessionStorage to boolean, but strings will definitely get it done 👍

File: src/SmartComponents/Recs/List.js [hunk 1]
```diff
@@ -39,14 +37,14 @@ const List = () => {
         <PageHeader className='ins-c-recommendations-header'>
             <PageHeaderTitle title={`${intl.formatMessage(messages.insightsHeader)} ${intl.formatMessage(messages.recommendations).toLowerCase()}`} />
             <DownloadExecReport />
-            {redhatDisabledRuleAlert && redHatDisabledRuleCount > 0 &&
+            {redhatDisabledRuleAlert === 'true' && redHatDisabledRuleCount > 0 &&
```
Comment (AllenBW) [inline src/SmartComponents/Recs/List.js:40]:
```suggestion
            {redhatDisabledRuleAlert && redHatDisabledRuleCount > 0 &&
```

agree with @LI0131 makin these bools means we can simplify this line

**Review by AllenBW — APPROVED** [review]
B.E.A.U.T.I.F.U.L.

File: src/SmartComponents/Recs/List.js (hunk 1 above)
Comment (mhuth) [reply src/SmartComponents/Recs/List.js]: Done!  Yep, bools look much better than strings.

**Comment by mhuth** [comment]
> Looking good @mhuth  You might consider using `JSON.parse` to deserialize the value from the sessionStorage to boolean, but strings will definitely get it done

Ah of course!  Thanks for the tip @LI0131

**Review by LI0131 — APPROVED** [review]
Awesome @mhuth 😁

**Pull request merged**